examples/CSharp/CSharp.Examples/Scenarios/DataFeed.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using NBomber;
+using NBomber.Contracts;
+using NBomber.CSharp;
+using Serilog;
+
+namespace CSharp.Examples
+{
+    class DataFeedScenario
+    {
+        public static void Run()
+        {
+            var feed =
+                Feed
+                    .Circular("numbers", Enumerable.Range(0, 10))
+                    .Select(x => (x + 1) * 10)
```

Comment:
it's like a map?

Reply:
Yes `Select` is here a C# alias for F# `map`

Reply:
ok